### ISSUE TITLE
Error processing callback: 'FetchResponse_v0' object is not subscriptable

### ISSUE THREAD

**Issue by zoltan-fedor** [dpkp/kafka-python#665]
I have just upgraded to v1.1.0 and receiving the following error when consuming a Kafka topic

```
ERROR:kafka.future:Error processing callback
Traceback (most recent call last):
  File "/data/apps/kabinet-dev/kabinet/venv1/lib64/python3.5/site-packages/kafka/future.py", line 33, in success
    f(value)
  File "/data/apps/kabinet-dev/kabinet/venv1/lib64/python3.5/site-packages/kafka/consumer/fetcher.py", line 656, in _handle_fetch_response
    self._sensors.fetch_throttle_time_sensor.record(response['throttle_time_ms'])
TypeError: 'FetchResponse_v0' object is not subscriptable
```

**Comment by dpkp** [comment]
Seems serious -- unclear why our tests pass. Investigating and expect to push a fix as 1.1.1 today.

**Comment by zoltan-fedor** [comment]
ok, looking forward to it.
I was upgrading last week to 1.0.2 and was also testing the dev version and the issue was already there then, but I thought the tests will catch it, so I haven't raised it - and then saw today getting released.
